Gopkg.toml

```diff
@@ -29,7 +29,7 @@ version="v1.4.7"
 
```

Comment:
Dependency merges are some of the nastiest merges I've seen. We should opt to upgrade in a commit before changes requiring those changes.

Reply:
I think this PR contains the small change needed for the dependency change (the use of KNative `validateContainer`). There are actually more changes due to the dependency update so this PR is really mainly the dependency update.

I have tested on a 0.5 KNative cluster.